common/persistence/src/main/java/app/coronawarn/datadonation/common/persistence/domain/metrics/ExposureRiskMetadata.java

```diff
@@ -19,23 +19,43 @@
   @Range(min = MIN_RISK_LEVEL, max = MAX_RISK_LEVEL,
       message = "Risk Level must be in between " + MIN_RISK_LEVEL + " and " + MAX_RISK_LEVEL + ".")
   private final Integer riskLevel;
+  /**
+   * The risk level reported by the client from check-in-based presence tracing.(0 to 3).
+   */
+  @Range(min = MIN_RISK_LEVEL, max = MAX_RISK_LEVEL,
```

Comment:
doesn't MAX_RISK_LEVEL need to be changed to 7 or 8?

Reply:
Same here :)